requirements-extras.txt

```
@@ -1,5 +1,5 @@
 qiskit-ibm-provider>=0.6.1 # for submitting experiments to backends through the IBM provider
 cvxpy>=1.3.2 # for tomography
 scikit-learn # for discriminators
-qiskit-aer>=0.11.0,<=0.12.2 # temporary version pin until 0.13.1 release
+qiskit-aer>=0.11.0
```

Should this be 
```suggestion
qiskit-aer>=0.11.0, !=0.13.0
```
to be safe? Or should we just raise the requirement to 0.13.1?

---

I put that at first but then took it out. The issues we were avoiding in 0.13.0 were Aer bugs (not treating reset as a default basis gate, changing the default simulation method leading to slower performance) that affected the tests. They were not actually issues with the experiments code. So I think the better argument would be to put `!=0.13.0` into `constraints.txt` but in practice it's probably not worth the effort since anyone updating Aer now should end up with 0.13.1 any way, so avoiding 0.13.0 is a pretty niche case. It wouldn't hurt much though (just maybe the extra overhead of someone in the future seeing that in constraints.txt and thinking about it before taking it out).